src/api/expenses.controller.js

```diff
@@ -0,0 +1,89 @@
+const { expensesService } = require('../services/expenses.service.js');
+const { usersService } = require('../services/users.service.js');
+
+const getExpenses = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = expensesService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
+    return res.sendStatus(400);
+  }
+
+  const user = usersService.getUserById(userId);
```

Comment:
Ensure that `usersService.getUserById` is correctly implemented and returns `null` or `undefined` when a user is not found. Otherwise, this check might not work as expected.